src/log.js

```diff
@@ -418,7 +418,7 @@ export class Log {
    *
    * Otherwise see `assert` for usage
    *
-   * @param {*} shouldBeElement
+   * @param {Element} shouldBeElement
```

Comment:
This in incorrect, the function can take any value

Reply:
in that case, a `?` might be most appropriate?